Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN touch /addons-server-docker-container \
         libffi-dev \
         libssl-dev \
         nodejs \
+        parallel \
```

Comment:
I like the PR, but `parallel` is a big heavy (at least the debian package is)...

```
# apt install parallel
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  cron exim4-base exim4-config exim4-daemon-light guile-2.2-libs libevent-2.1-6 libgc1c2 libgnutls-dane0 libgsasl7 libkyotocabinet16v5 libltdl7 liblzo2-2 libmailutils5 libmariadb3 libntlm0
  libprocps7 libpython2.7 libpython2.7-minimal libpython2.7-stdlib libsensors-config libsensors5 libunbound8 libwrap0 mailutils mailutils-common mariadb-common procps psmisc sysstat
Suggested packages:
  anacron logrotate checksecurity exim4-doc-html | exim4-doc-info eximon4 spf-tools-perl swaks dns-root-data lm-sensors mailutils-mh mailutils-doc isag
The following NEW packages will be installed:
  cron exim4-base exim4-config exim4-daemon-light guile-2.2-libs libevent-2.1-6 libgc1c2 libgnutls-dane0 libgsasl7 libkyotocabinet16v5 libltdl7 liblzo2-2 libmailutils5 libmariadb3 libntlm0
  libprocps7 libpython2.7 libpython2.7-minimal libpython2.7-stdlib libsensors-config libsensors5 libunbound8 libwrap0 mailutils mailutils-common mariadb-common parallel procps psmisc
  sysstat
0 upgraded, 30 newly installed, 0 to remove and 0 not upgraded.
```

Could we find something lighter that doesn't force us to pull all these packages ?

Reply:
Simplest option might be to use `xargs` with a fixed number of processes that makes sense for our CI environment (4 ?)

Reply:
https://www.gnu.org/software/parallel/parallel_alternatives.pdf lists some alternatives - we don't need all the extensive feature set it supports